### ISSUE TITLE
Keyboard support

### ISSUE THREAD

**Issue by orosbogdan** [craftworkgames/MonoGame.WpfCore#15]
I think keyboard support together with a sample would help a lot of users.

**Comment by Rob-Storm** [comment]
I agree, having to use the Windows.Input workaround is a little antithetical to using this library
